sdk/nodejs/runtime/template/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "template",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "tsx": "^4.6.2",
+    "typescript": "^5.3.2"
+  },
+  "devDependencies": {
+    "@lifeomic/axios-fetch": "^3.0.1",
+    "adm-zip": "^0.5.10",
+    "env-paths": "^3.0.0",
+    "execa": "^8.0.1",
+    "graphql": "^16.8.0",
+    "graphql-request": "^6.1.0",
+    "graphql-tag": "^2.12.6",
+    "node-color-log": "^10.0.2",
+    "node-fetch": "^3.3.1",
+    "tar": "^6.1.13"
+  }
+}
```

Comment:
Why are all of these needed on the user's module? There's ways to provide dependency groups even if you make a _meta_ package in the sdk itself. It would install everything needed for modules and the library as well.

Reply:
Because if I load dagger as a meta package, the registry will be empty since the one used by the user's module will be the one from `dist`, but the one using in the entrypoint will be the one from actual sources.
It also simplify the process because I don't have to build the SDK to install it.

Basically, without this, it cannot work, that's why I kept them in the dev dependencies

Reply:
I think we can make it work in a way that you don't need these dependencies on the user's side. You may be complicating if you're getting different registry instances. I think that can be fixed in a simple way. Doesn't need to block the merge though. Can be in a follow up.